src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/JenkinsBuildLogParseUtils.java

```diff
@@ -0,0 +1,81 @@
+package de.tum.in.www1.artemis.service.connectors.jenkins;
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
+import org.jsoup.nodes.Element;
+import org.jsoup.nodes.Node;
+import org.jsoup.nodes.TextNode;
+
+import de.tum.in.www1.artemis.domain.BuildLogEntry;
+
+public class JenkinsBuildLogParseUtils {
+
+    // Pattern of the DateTime that is included in the logs received from Jenkins
+    private static final DateTimeFormatter LOG_DATA_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ssX");
+
+    public static List<BuildLogEntry> parseBuildLogsFromJenkinsLogs(List<String> logLines) {
```

Comment:
It would be nice to also have some _JavaDoc_ comments on this method. You could add here, also what the log coming from Jenkins looks like, since it's difficult to extract from the code, what the input being parsed could look like.